src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -200,7 +199,42 @@ public async Task<string> SubmitPledge(CreatePostRequest request)
 
             return _encodingService.Encode(pledgeId, EncodingType.PledgeId);
         }
+        public async Task RejectApplications(RejectApplicationsPostRequest request)
+        { 
+            var serviceRequest = new SetRejectApplicationsRequest
+            {
+                UserId = _userService.GetUserId(),
+                UserDisplayName = _userService.GetUserDisplayName(),
+            };
+            serviceRequest.ApplicationsToReject.AddRange(from applicationId in request.ApplicationsToReject
```

Comment:
We don't tend to use this style of LINQ elsewhere, so could we avoid it? I also dislike creating an object in one statement, then setting some of its properties in subsequent statements; could they be combined?

Reply:
Yes sure, I have removed the LINQ and refactored that method.